src/GrpcService.ts

```diff
@@ -1,14 +1,11 @@
 import * as semver from 'semver';
 
-// If user is not using Node.js v10, warn to upgrade to v10
-if (!semver.satisfies(process.version, '10.x')) {
-    let errorMessage = `Your Function App is currently set to use Node.js version ${process.version}, but the runtime requires a 10.x version.
+// If user is not using Node.js v8 or v10, fail
+if (!(semver.satisfies(process.version, '8.x') || semver.satisfies(process.version, '10.x'))) {
+    let errorMessage = `Your Function App is currently set to use Node.js version ${process.version}, but the runtime requires an Active LTS or Current version (ex: 8.11.1 or 10.6.0).

```

Comment:
Delete - or current version

Reply:
@pragnagopa - "Current" is a node.js release term which captures 10.x versions, i think we should keep it..!
https://github.com/nodejs/Release